convert2rhel/actions/pre_ponr_changes/backup_system.py

```diff
@@ -75,28 +75,30 @@ class BackupRepository(actions.Action):
     id = "BACKUP_REPOSITORY"
 
     def run(self):
-        """Backup repository files before starting conversion process"""
+        """Backup .repo files in /etc/yum.repos.d/ so the repositories can be restored on rollback."""
         loggerinst.task("Prepare: Backup Repository Files")
 
         super(BackupRepository, self).run()
 
-        self.backup_yum_repos()
-
-    def backup_yum_repos(self):
-        """Backup .repo files in /etc/yum.repos.d/ so the repositories can be restored on rollback."""
         loggerinst.info("Backing up .repo files from %s." % DEFAULT_YUM_REPOFILE_DIR)
 
-        repo_files_backed_up = False
+        if os.listdir(DEFAULT_YUM_REPOFILE_DIR) == []:
+            loggerinst.info("No .repo files to back up.")
```

Comment:
What if we used a different message? Maybe something like: 

```suggestion
            loggerinst.info("Repository folder %s seems to be empty.", DEFAULT_YUM_REPOFILE_DIR)
```

Reply:
Just to differentiate from the other log message when a file does not end in .repo